src/Sarif.Driver.UnitTests/AnalyzeCommandTests.cs

```diff
@@ -308,10 +308,10 @@ public void IOExceptionRaisedCreatingSarifLog()
         [Fact]
         public void UnauthorizedAccessExceptionCreatingSarifLog()
         {
-            string path = Environment.GetFolderPath(Environment.SpecialFolder.Windows);
+            string path = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
             path = Path.Combine(path, Guid.NewGuid().ToString());
```

Comment:
Oh, I see. I had to look at the whole file to understand that the failure happens when you try to persist to a file that already exists. And the change to ApplicationData is just because that's a better place than littering Windows.